.travis.yml

```diff
@@ -7,3 +7,6 @@ rvm:
   - 2.5
   - 2.4
   - 2.3
+script:
+  - bundle exec rubocop
```

Comment:
No need, it's covered by simple `rake` - see https://github.com/bsm/sidekiq-datadog/blob/master/Rakefile#L23

Reply:
My bad